templates/facets/block--facets.html.twig

```diff
@@ -0,0 +1,10 @@
+<div{{ attributes }}>
```

Comment:
not related to this template but was thinking of moving all templates from under /overrides into /templates